DigitalLearningSolutions.Web/Styles/trackingSystem/centreRanking.scss

```diff
@@ -0,0 +1,21 @@
+﻿@import "~nhsuk-frontend/packages/core/all";
+
+$current-centre-highlight: $color_nhsuk-pale-yellow;
+
+.current-centre {
+  background-color: $current-centre-highlight;
+
+  &:hover {
+    background-color: $current-centre-highlight;
+  }
+}
+
+.cell-left-padding {
+  padding-left: nhsuk-spacing(2);
+}
+
+.cell-right-padding {
+  @include govuk-media-query($until: desktop) {
+    padding-right: nhsuk-spacing(2) !important;

```

Comment:
It gets overwritten by nhs styles .nhsuk-table-responsive .nhsuk-table__body .nhsuk-table__row td without the !important

Reply:
That is quite high specificity.. Slightly unrelated, but does using the NHS padding classes instead of these like we do for margins not work in this case?

Reply:
That would apply padding for the layout all the way to desktop sizes, this class only applies it when they are collapsed in the tablet/mobile layout